docs/source/contents/4_draft_mapping.rst

```diff
@@ -3045,9 +3046,9 @@ and `archivision:1754 <https://digital.lib.utk.edu/collections/islandora/object/
 
 .. code-block:: turtle
 
-    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+    @prefix dcterms: <http://purl.org/dc/terms/> .
 
-    <https://example.org/object/1> edm:hasType <http://id.loc.gov/authorities/subjects/sh85040974> .
```

Comment:
Addresses: https://hyp.is/1XcD7hgeEe2NlwPyO2pP9Q/utk-mods-to-rdf.readthedocs.io/en/latest/contents/4_draft_mapping.html